src/supabase/queries/auth.ts

```diff
@@ -57,3 +51,45 @@ export const signOutUser = async () => {
     throw error;
   }
 };
+
+export const deleteCurrentUser = async (userId: string) => {
```

Comment:
Let's add `console.warn` statements to all of the errors in the try-catches for consistency!